scripts/update-sdk.sh

```diff
@@ -17,9 +17,12 @@
 set -euo pipefail
 
 LATEST_SDK_TAG=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')
-
 LATEST_SDK_RELEASE=$(echo "${LATEST_SDK_TAG}" | cut -d '.' -f 1)
-echo  "==> Updating SDK to latest major version ${LATEST_SDK_TAG}"
+
```

Comment:
Could you clarify what was the issue and what the improvement should help with? Thanks!

Reply:
as an example a new Atlas Go SDK version was released this morning, but it took many hours until the script was successful because Go caches.  We want to try to avoid/minimize that delay